specs/source/management/identities.yml

```diff
@@ -355,6 +355,35 @@ paths:
           $ref: '../shared/standard-responses.yml#/responses/notFoundResponse'
         '401':
           $ref: '../shared/standard-responses.yml#/responses/unauthorizedResponse'
+  identities-id-trace:
+    parameters:
+      - $ref: '../shared/parameters.yml#/id'
+    put:
+      summary: Enable/disable data flow tracing for an identity
+      description: |
+        Allows an admin to enable/disable data flow tracing for an identity
+      security:
+        - ztSession: [ ]
+      tags:
+        - Identity
+        - Tracing
+      operationId: updateIdentityTracing
+      parameters:
+        - name: traceSpec
+          in: body
+          required: true
+          description: A traceSpec object
+          schema:
+            $ref: '#/definitions/traceSpec'
+      responses:
+        '200':
+          $ref: '#/responses/traceDetail'
```

Comment:
For the rest of the API a `PUT` to a document URL returns an `emptyResponse` from the shared responses.

Reply:
I see you are returning a different object with a calculated until value.

Reply:
Yes, also if you don't provide a traceId, it will generate one for you.